### ISSUE TITLE
trash-restore dose not exist from a apt package

### ISSUE THREAD

**Issue by AStupidBear** [andreafrancia/trash-cli#172]
After installing `trash-cli` using `sudo apt-get install trash-cli`, I missed `trash-restore` commond.  Is this a mistake?

**Comment by rk200525** [comment]
i installed using `sudo apt-get install trash-cli` on 628su, did not find any issue with `trash-restore`

**Comment by jmtd** [comment]
Debian (co)maintainer here. The package contains `/usr/bin/trash-restore`. See e.g. https://packages.debian.org/sid/all/trash-cli/filelist to confirm this.

**Comment by pedrovagner** [comment]
I'm facing this same problem, missing `trash-restore` command. Installed using `sudo apt install trash-cli`. More info:

```shell
$ trash --version
trash 0.12.9.14

OS: Kubuntu 18.04
```

**Comment by jmtd** [comment]
Thanks for clarifying which OS you are talking about. You'd have to check with the Raspbian folks or Ubuntu on their policy for updating those releases, although at least in the case of Raspbian, that version is very out-of-date. (FWIW Debian's jessie package has the same bug, but Jessie is long out of support now, so it won't be fixed there)

**Comment by AStupidBear** [comment]
I'm on ubuntu 18.04.

**Comment by AStupidBear** [comment]
https://github.com/andreafrancia/trash-cli/issues/172#issuecomment-667091040 solved my problem

**Comment by pedrovagner** [comment]
I just found out `restore-trash` installed in my machine. I suppose there is a version confusion between `trash-restore` and `restore-trash`. Can someone confirm this fact?

**Comment by AStupidBear** [comment]
@pedrovagner I can confirm this

**Comment by andreafrancia** [comment]
In version 0.16.12.25 restore-trash was renamed trash-restore.
The version number used in trash-cli follow the pattern 0.YY.MM.DD. The version 0.16.12.25 has been published on Dec 25 2016.
Unfortunately some linux distributions include very old versions of trash-cli.
This is the list of changes and bug fix since the version 0.12.9.14

```
0.20.11.14:
 - switched to psutil for listing volumes

0.20.11.7:
 - trash-put learned a --trash-dir option that can be used to specify the trash
   directory to be used as destination 
 - trash-put -f now ignores files and dir that does not exist (Don Cross)
 - trash-restore learnt --sort=(date|path|none) option (Self-Perfection)
 - trash-restore: support restoring multiple files (arendu)
 - README: now recommend using `pip` for installing trash-cli

0.17.1.14:
 - Fix a bug that causes trash-put to use $topdir/.Trash/UID trashcan even when
   it is not secure and $topdir/.Trash-UID should be used instead. 

0.17.1.12:
 - Fix a bug in detecting right volume of home trash dir home Trash dir is a
   symbolic link, and in detecting volume of file to be trashed when it is
   specified as contained in a directory that is a symbolic link that crosses
   volumes boundaries (#38)
 - Make some code  python 3 compatible 
 - Fixed README.
 
0.17.1.1:
 - Now trash-rm supports full path matching, using a pattern starting with slash '/' Fix #67
 - Fix typo in trash-rm(1) man page
 - Add a reference to trash-rm(1) to all man pges
 - Fix inconsistent apostrophes
 - Add support for --trash-dir option to trash-empty

0.16.12.29:
 - trash-rm no more crashes on .trashinfo files without Path (#69)

0.16.12.28:
 - Fix #48 trash-empty crashes on directories without read permission

0.16.12.26:
 - Fix #52: Almost all commands crashes with python 2.7.10

0.16.12.25:
 - Now trash-restore accept a /specific/path
 - Now integration tests should pass also in a linux box. Fix #61
 - Now all commands output will report the right up-to-date location for issue reporting (#39)
 - Add input validation in trash-restore
 - Renamed trash-restore instead of restore-trash
 - Fixed bug (trash-put creates $topdir/.Trash even it should not)
 - Fixed bug (trash-put uses $topdir/.Trash/$uid even if unsecure)
 - Minor changes to man pages.

```

My suggestion is to remove the distribution version and install a newer version from pypi:
```
sudo apt-get remove trash-cli
sudo pip install trash-cli
```
